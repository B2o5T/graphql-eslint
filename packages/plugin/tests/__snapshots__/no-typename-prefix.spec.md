// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invalid #1 1`] = `
#### ⌨️ Code

      1 |         type User {
      2 |           userId: ID!
      3 |         }

#### ❌ Error

      1 |         type User {
    > 2 |           userId: ID!
        |           ^^^^^^ Field "userId" starts with the name of the parent type "User"
      3 |         }

#### 💡 Suggestion: Remove \`user\` prefix

    1 |         type User {
    2 |           Id: ID!
    3 |         }
`;

exports[`Invalid #2 1`] = `
#### ⌨️ Code

      1 |         type User {
      2 |           userId: ID!
      3 |           userName: String!
      4 |         }

#### ❌ Error 1/2

      1 |         type User {
    > 2 |           userId: ID!
        |           ^^^^^^ Field "userId" starts with the name of the parent type "User"
      3 |           userName: String!

#### 💡 Suggestion: Remove \`user\` prefix

    1 |         type User {
    2 |           Id: ID!
    3 |           userName: String!
    4 |         }

#### ❌ Error 2/2

      2 |           userId: ID!
    > 3 |           userName: String!
        |           ^^^^^^^^ Field "userName" starts with the name of the parent type "User"
      4 |         }

#### 💡 Suggestion: Remove \`user\` prefix

    1 |         type User {
    2 |           userId: ID!
    3 |           Name: String!
    4 |         }
`;

exports[`Invalid #3 1`] = `
#### ⌨️ Code

      1 |         interface Node {
      2 |           nodeId: ID!
      3 |         }

#### ❌ Error

      1 |         interface Node {
    > 2 |           nodeId: ID!
        |           ^^^^^^ Field "nodeId" starts with the name of the parent type "Node"
      3 |         }

#### 💡 Suggestion: Remove \`node\` prefix

    1 |         interface Node {
    2 |           Id: ID!
    3 |         }
`;
