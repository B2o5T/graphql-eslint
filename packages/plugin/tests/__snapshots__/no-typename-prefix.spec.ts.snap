// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Error 1/1

  1 |
  2 |         type User {
> 3 |           userId: ID!
    |           ^^^^ Field "userId" starts with the name of the parent type "User"
  4 |         }
  5 |       
`;

exports[` 2`] = `
Error 1/2

  1 |
  2 |         type User {
> 3 |           userId: ID!
    |           ^^^^ Field "userId" starts with the name of the parent type "User"
  4 |           userName: String!
  5 |         }
  6 |       

Error 2/2

  1 |
  2 |         type User {
  3 |           userId: ID!
> 4 |           userName: String!
    |           ^^^^ Field "userName" starts with the name of the parent type "User"
  5 |         }
  6 |       
`;

exports[` 3`] = `
Error 1/1

  1 |
  2 |         interface Node {
> 3 |           nodeId: ID!
    |           ^^^^ Field "nodeId" starts with the name of the parent type "Node"
  4 |         }
  5 |       
`;
